### ISSUE TITLE
APIs broken?

### ISSUE THREAD

**Issue by goldsborough** [DataUSA/datausa-api#69]
Hi,

Was really excited to use the DataUSA API, but all the APIs mentioned in the docs seem broken. Anything hitting `http://api.datausa.io` is giving me
```
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.10.3 (Ubuntu)</center>
</body>
</html>
```

and looking at the API mentioned at https://datausa.io/about/api, the only thing I can get done is

```
curl https://datausa.io/api/data\?drilldowns\=Place\&measures\=Population\&year\=latest
```

but then there's no further indication of how to do anything else with this URL (i.e. what other "measures" are there)?

The site [here](https://datausa.io/about/api/) also says:
> All of the visualizations on the page have a "show data" button on their top-right that displays the API call(s) used to generate that visualization

but didn't find any such button. 

So bottom line, unable to use your API, sadly. Any simple fixes here?

**Comment by davelandry** [comment]
@goldsborough apologies for not providing up-to-date API documentation. The legacy API at `api.datausa.io` was taken down a few months ago, so that explains the 301 errors 😢 

Using `https://datausa.io/api/data` is the preferred method as of right now, and is what can be seen under the "view data" button to the top-right of visualizations (sorry about the incorrect "show" language on that page, we'll get that updated):

<img width="1208" alt="Screen Shot 2021-03-22 at 4 32 24 PM" src="https://user-images.githubusercontent.com/1847581/112054634-304b5f80-8b2c-11eb-9d82-c6ead0165507.png">

In the meantime, if you are looking for a specific data query, I can help you find that (just let me know). If you want to dig into what's available (it's a lot), this endpoint contains a JSON result with a list of every available measure and their drilldowns (along with some other behind the scenes stuff): https://datausa.io/api/cubes
